app/models/user.rb

```diff
@@ -55,6 +57,7 @@ class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
   validates :level, :specialty, presence: true, if: :engineer?
   validates :github, uniqueness: true, if: :engineer?
+  validates :roles, presence: true
```

Comment:
now we have multiple roles we don't need this to be required, if the user has no roles, this will be the normal